client/config/environment.js

```diff
@@ -37,17 +38,15 @@ module.exports = function(environment) {
     ENV.contentSecurityPolicy = {
       'default-src': "'none'",
       // add the domain here like analytics.google.com or data: or 'self'
-      'script-src':  "'self' 'unsafe-eval' 'https://www.google.com' ''",
+      'script-src':  "'self' 'unsafe-eval' 'unsafe-inline' www.google.com www.google-analytics.com",
```

Comment:
I just tested it out without having `'unsave-inline'` specified and did not get any security policy warnings so I'm thinking we can move forward with these changes with out it.  We can always add it back if we determine there are issues once these changes are deployed.